Marlin/Configuration_adv.h

```diff
@@ -39,6 +39,23 @@
 //=============================Thermal Settings  ============================
 //===========================================================================
 
+// hotend thermistor parameters (type -999)
+#define HOTEND0_NTC_SERIES_RESISTOR_OHMS     4700    // ntc pull-up resistor
+#define HOTEND0_NTC_RESISTANCE_25C_OHMS      100000  // ntc resistance at 25C
+#define HOTEND0_NTC_BETA                     3950    // ntc beta value
+
+// .. add the required hotend defines (1 to 5) if you specify thermistor type -999 in Configuration.h
+
+// bed thermistor parameters (type -999)
+#define BED_NTC_SERIES_RESISTOR_OHMS         4700    // ntc pull-up resistor
+#define BED_NTC_RESISTANCE_25C_OHMS          100000  // ntc resistance at 25C
+#define BED_NTC_BETA                         3950    // ntc beta value
+
+// chamber thermistor parameters (type -999)
+#define CHAMBER_NTC_SERIES_RESISTOR_OHMS     4700    // ntc pull-up resistor
+#define CHAMBER_NTC_RESISTANCE_25C_OHMS      100000  // ntc resistance at 25C
+#define CHAMBER_NTC_BETA                     3950    // ntc beta value
```

Comment:
I assume these are the names intended for configuration, yes? If so then I can replace `HOTEND0_THERMISTOR_SERIES_RESISTOR_OHMS` with `HOTEND0_NTC_SERIES_RESISTOR_OHMS` (etc.) throughout the code.

Reply:
Sure if you like, no problem.

Reply:
My main concern up to now has just been creating the code and debugging it. The names I've swapped and urmed on whilst I've been going through it.

Reply:
Your **log_user_thermistor()** function in M305.cpp might be better to not be inline'd, you'd save flash space that way, just having the one defined function rather than repeated.

Reply:
The coding style is good. There are a few places where Marlin has some helpful resources and guidelines, so I'm just applying those where needed at the moment….